lib/init.js

```diff
@@ -1,30 +1,26 @@
-var readline = require('readline')
-var path = require('path')
-var glob = require('glob')
-var mm = require('minimatch')
-var exec = require('child_process').exec
+'use strict'
 
-var helper = require('./helper')
-var logger = require('./logger')
+const readline = require('readline')
```

Comment:
We can consider using of [inquirer](https://github.com/SBoudrias/Inquirer.js). It contains set of useful prompts, and has nice questions API.